tests/test_features/test_jointplot.py

```diff
@@ -0,0 +1,65 @@
+# tests.test_features.test_jointplot
+# Test the JointPlotVisualizer
+#
+# Author:   Prema Damodaran Roman
+# Created:  
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: test_jointplot.py
+
+"""
+Test the JointPlotVisualizer
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import unittest
+import numpy as np
+import numpy.testing as npt
+
+from tests.dataset import DatasetMixin
+from yellowbrick.features.jointplot import *
+
+##########################################################################
+## JointPlotVisualizer Tests
+##########################################################################
+
+class JointPlotTests(unittest.TestCase, DatasetMixin):
+
+	X = np.array([1, 2, 3, 5, 8, 10])
+
+	y = np.array([1, 3, 6, 2, 9, 2])
+	
+	def setUp(self):
+		self.concrete = self.load_data('concrete')
+
+	def tearDown(self):
+		self.concrete = None
+
+	def test_jointplot(self):
+		"""
+		Assert no errors occur during jointplot visualizer integration
+		"""
+		
+		visualizer = JointPlotVisualizer()
+		visualizer.fit(self.X, self.y)
```

Comment:
I would also call `visualizer.poof()` under this line like you're doing in the text test.